test/unstable/externProcArrayFormal.compopts

```diff
@@ -0,0 +1 @@
+--no-codegen
```

Comment:
Did this allow you to avoid including a `.h` and/or `.c` file?  Or did you forget to commit those?

Reply:
Yes, the check for an existing C function happens in codegen. It is also why there is a `noexec`.